### ISSUE TITLE
Unknown argument "handle" on directive "@imagerTransform"

### ISSUE THREAD

**Issue by jonnyasmar** [spacecatninja/craft-imager-x#126]
### I'm submitting a... 

- [x] bug report
- [ ] feature request
- [x] general question

### Steps to reproduce

1. Use the @imagerTransform directive on a GraphQL query
2. Use the "handle" argument to select a transform from preconfigured transforms in `config/imager-x-transforms.php`
3. Observe the error in the GraphQL response

e.g.

```gql
image {
  url @imagerTransform(handle: "someTransform")
} 
```

### Description
 

I was using this method previously with v3.2.0 to generate transforms, but after running into some issues where files would intermittently and sporadically become unable to be opened by imager (causing transforms to fail; separate bug) I attempted to upgrade Imager X to the latest version and began seeing this issue. Now, even after downgrading back to 3.2.0, I am still experiencing the issue. It appears the @imagerTransform directive doesn't accept the handle argument anymore, but I'm confused how this would still be the case after downgrading.

In the process, I did have to upgrade Craft from 3.5.13 to 3.7.21, but I've also downgraded back to 3.5.13 to no avail.

### Displayed errors, stack trace, relevant logs  

*See issue title*

### Additional info

- Imager version & edition: 3.2.0 -> 3.5.4 -> 3.2.0
- Imager transformer: Craft
- Craft version: 3.5.13 -> 3.7.21 -> 3.5.13
- PHP version: 7.2.24
- Image driver & version: 
- Database driver & version:
- Plugins & versions (if relevant):

**Comment by jonnyasmar** [comment]
Hmm, I seem to have got this to work again by making a small change to `src/gql/directives/ImagerTransform.php`. Added this after line 88 as well as the corresponding argument to `src/gql/arguments/ImagerTransformArguments.php`:

```php
        if (isset($arguments['handle'])) {
            $transform = $arguments['handle'];
        }
```

I wonder if I had done this in the past and forgot about it (it's been a very long time since I touched this area of our app).

I would provide a pull request for this change, but I've just moved and had a baby so it's unlikely I'll have the time to dive in and update documentation, tests, etc. Would love to see this make it's way into the library, though! Imager X w/ GraphQL is awesome :) If I have time in the future and you haven't got around to it, I will definitely contribute a PR.

Thanks! Feel free to close this issue unless you have more to add.

**Comment by aelvan** [comment]
Named transforms (ie `handle`) were never supported by `@imagerTransform`, so I think you must've hacked it in there in the first place. :)

I mulled over whether or not to add support for it. The issue being that I'd assumed that people usually define named transformed for an array of transforms, rather that a single transform. The directive can only return a string, not an array of strings, so the named transform handle you pass in can only be for a single transform.

In the end I decided that I'll just add it and hope it won't cause too much confusion. If the user passes in a named transform that results in an array, it'll return null and log an error message. Hopefully people will pick up on that.

Just to be clear what I mean, for anyone landing here. This named transform will work with `@imagerTransform`:

```
'myTransform' => [
    'transforms' => [
         'width' => 600
    ]
],
```

This will NOT:

```
'myTransform' => [
    'transforms' => [
        ['width' => 600],
        ['width' => 900],
        ['width' => 1200],
    ]
],
```
